INFO:     Will watch for changes in these directories: ['/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi']
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [54505] using StatReload
Process SpawnProcess-1:
Traceback (most recent call last):
  File "/Users/sprague/miniconda3/lib/python3.10/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/Users/sprague/miniconda3/lib/python3.10/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/venv/lib/python3.10/site-packages/uvicorn/_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/venv/lib/python3.10/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/Users/sprague/miniconda3/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/Users/sprague/miniconda3/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/venv/lib/python3.10/site-packages/uvicorn/server.py", line 70, in serve
    await self._serve(sockets)
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/venv/lib/python3.10/site-packages/uvicorn/server.py", line 77, in _serve
    config.load()
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/venv/lib/python3.10/site-packages/uvicorn/config.py", line 435, in load
    self.loaded_app = import_from_string(self.app)
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/venv/lib/python3.10/site-packages/uvicorn/importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
  File "/Users/sprague/miniconda3/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/app.py", line 12, in <module>
    from weasyprint import HTML
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/venv/lib/python3.10/site-packages/weasyprint/__init__.py", line 430, in <module>
    from .css import preprocess_stylesheet  # noqa: I001, E402
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/venv/lib/python3.10/site-packages/weasyprint/css/__init__.py", line 28, in <module>
    from .computed_values import COMPUTER_FUNCTIONS
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/venv/lib/python3.10/site-packages/weasyprint/css/computed_values.py", line 9, in <module>
    from ..text.ffi import FROM_UNITS, ffi, pango
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/venv/lib/python3.10/site-packages/weasyprint/text/ffi.py", line 475, in <module>
    gobject = _dlopen(
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/venv/lib/python3.10/site-packages/weasyprint/text/ffi.py", line 463, in _dlopen
    return ffi.dlopen(names[0], flags)  # pragma: no cover
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/venv/lib/python3.10/site-packages/cffi/api.py", line 150, in dlopen
    lib, function_cache = _make_ffi_library(self, name, flags)
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/venv/lib/python3.10/site-packages/cffi/api.py", line 834, in _make_ffi_library
    backendlib = _load_backend_lib(backend, libname, flags)
  File "/Users/sprague/dev/proofbound/customer-books/bookgeneratorapi/venv/lib/python3.10/site-packages/cffi/api.py", line 829, in _load_backend_lib
    raise OSError(msg)
OSError: cannot load library 'libgobject-2.0-0': dlopen(libgobject-2.0-0, 0x0002): tried: 'libgobject-2.0-0' (no such file), '/System/Volumes/Preboot/Cryptexes/OSlibgobject-2.0-0' (no such file), '/Users/sprague/miniconda3/bin/../lib/libgobject-2.0-0' (no such file), '/usr/lib/libgobject-2.0-0' (no such file, not in dyld cache), 'libgobject-2.0-0' (no such file).  Additionally, ctypes.util.find_library() did not manage to locate a library called 'libgobject-2.0-0'

-----

WeasyPrint could not import some external libraries. Please carefully follow the installation steps before reporting an issue:
https://doc.courtbouillon.org/weasyprint/stable/first_steps.html#installation
https://doc.courtbouillon.org/weasyprint/stable/first_steps.html#troubleshooting 

-----

INFO:     Stopping reloader process [54505]
